modules/olm-about-catalogs.adoc

```diff
@@ -26,3 +26,4 @@ Support for the legacy _package manifest format_ for Operators, including custom
 
 When creating custom catalog images, previous versions of {product-title} 4 required using the `oc adm catalog build` command, which was deprecated for several releases and is now removed. With the availability of Red Hat-provided index images starting in {product-title} 4.6, catalog builders must use the `opm index` command to manage index images.
 ====
+//How many versions do we need to pull this note forward?
```

Comment:
@adellape, PTAL?

Reply:
I feel like since this format change from pkg man format -> bundle format just happened in 4.8, and we're also now introducing the new file-based catalog format in 4.9 (which is in the same wheelhouse), I'd like to keep this note around at least until 4.10 to avoid confusion, and then re-evaluate. Keeping these deprecated/removed "formats" straight is getting to be a juggling act. :dizzy:

Reply:
I'll tag it so we think about it again during the 4.10 to 4.11 scrub. Thanks!